stix_shifter/stix_translation/src/utils/transformers.py

```diff
@@ -116,7 +117,7 @@ class ToFilePath(ValueTransformer):
     @staticmethod
     def transform(obj):
         try:
-            return obj[0:len(obj) - len(re.split(r'[\\/]', obj)[-1])]
+            return os.path.dirname(obj)
```

Comment:
not sure why you changed to `os.path.dirname(obj)`. i dont see you get correct file path and also its breaking some other module path transformation.

Reply:
This was added to get directory path without trailing "\" or "/" slash. I have updated other impacted tests cases in my earlier commit. I have added new transformer function for bigfix and msatp implementation and reverted the other test changes.